docs/CHANGELOG.md

```diff
@@ -36,6 +36,8 @@ it is attached to. ([#1282])
 * Adding support for AMETHYST_NUM_THREADS environment variable to control size of the threads pool used by thread_pool_builder.
 * Add `Input` variant to `StateEvent`. ([#1478])
 * Support type parameters in `EventReader` derive. ([#1478])
+* Added `monitor_width` and `monitor_height` in `ScreenDimensions`. ([#1539])
+* Added `centered_window` bool to `DisplayConfig` Windows will now be centered by deafult. ([#1539])
```

Comment:
Please add the link to the pull request. See the lines under those ones to see how.